aqua-examples/ts-oracle/Readme.md

```diff
@@ -6,98 +6,153 @@ Getting accurate timestamps can be problematic in various contexts including blo
 
 **Note: The timestamps are currently** **not** **signed by the nodes. But that could be easily addressed.**
 
+For this tutorial, we'll be using Fluence's new `fluence` CLI tool, which wraps the CLIs you have already been using, e.g., the `aqua` CLI and Marine tooling CLIs (`marine` and `mrepl`), and brings additional features such as project template generation, wrapper generation for deployed services, project dependencies install. See the [Fluence CLI docs](https://github.com/fluencelabs/fluence-cli#readme) for more information.
+
 ### Fluence Solution
 
 Fluence provides an open Web3 protocol, framework and associated tooling to develop and host applications, interfaces and backends on permissionless peer-to-peer networks. An integral part of the Fluence solution is the Aquamarine stack comprised of Aqua and Marine. Aqua is a new programming language and paradigm purpose-built to program distributed networks and compose applications from distributed services. For more information on Aqua, see
 
-* [Aqua Book](https://app.gitbook.com/@fluence/s/aqua-book/)
+* [Aqua Book](https://doc.fluence.dev/aqua-book/)
 * [Aqua Playground](https://github.com/fluencelabs/aqua-playground)
 * [Aqua repo](https://github.com/fluencelabs/aqua)
 
 
-Marine is a general purpose Wasm runtime and toolkit, allows developers to build distributed services that can be composed into distributed applications by Aqua. For more information on Marine, see
+Marine is a general-purpose Wasm runtime and toolkit, it allows developers to build distributed services that can be composed into distributed applications by Aqua. For more information on Marine, see
 
+* [Marine Book](https://doc.fluence.dev/marine-book/)
 * [Marine repo](https://github.com/fluencelabs/marine)
 * [Marine SDK](https://github.com/fluencelabs/marine-rs-sdk)
 
 ### Setup
 
-*Note that we already deployed the service to node `12D3KooWHLxVhUQyAuZe6AHMB29P7wkvTNMn7eDMcsqimJYLKREf` with service id `ed657e45-0fe3-4d6c-b3a4-a2981b7cadb9`, which is all what's needed to use the service in Aqua.*
-
-In order to run the entire code base, Rust and Node required. If necessary see [Install Rust](https://www.rust-lang.org/tools/install) and [NVM](https://github.com/nvm-sh/nvm) for details.
+In order to run the entire code base, Rust and Node are required. If necessary see [Install Rust](https://www.rust-lang.org/tools/install) and [NVM](https://github.com/nvm-sh/nvm) for details.
 
-Install the following tools:
+Install the Fluence CLI by running the following command:
 
 ```bash
-cargo install mrepl 
-cargo install marine
-
-npm -g install @fluencelabs/aqua
+npm install -g @fluencelabs/cli
 ```
 
-To compile the code to the wasi target:
+You can use the REPL to locally test the services:
 
 ```bash
-./scripts/build.sh
+fluence service repl tsOracle
```

Comment:
looks to me all the rust code organization and Cargo.toml are outdated since the setup is from my cargo generate template.

Reply:
Do you mean SDKs versions? For me it seemed one from the CLI is way older in terms of versions.

That's what I got upon generation

```
[package]
name = "newService"
version = "0.1.0"
edition = "2018"

[[bin]]
name = "newService"
path = "src/main.rs"

[dependencies]
marine-rs-sdk = "0.6.15"

[dev-dependencies]
marine-rs-sdk-test = "0.5.0"
```

It seems it requires some update in the CLI.